src/data/matic/curvePools.json

```diff
@@ -151,5 +151,10 @@
         "decimals": "1e18"
       }
     ]
+  },
+  {
+    "name": "crv-mai-3pool",
```

Comment:
lets rename to `curve-mai-3pool`, and please help to specify tokens array as we need it for incoming refactoring https://github.com/beefyfinance/beefy-api/blob/master/src/data/matic/curvePools.json#L141
seems it should be exactly same eurt, just different oracleId and decimals